beacon-chain/node/node.go

```diff
@@ -246,7 +248,21 @@ func (b *BeaconNode) registerPOWChainService(cliCtx *cli.Context) error {
 	depAddress := cliCtx.GlobalString(utils.DepositContractFlag.Name)
 
 	if depAddress == "" {
-		log.Fatal("No deposit contract specified. Add --deposit-contract with a valid deposit contract address to start.")
+		resp, err := http.Get("https://beta.prylabs.net/contract")
```

Comment:
```suggestion
              log.Info("Fetching testnet cluster address from https://beta.prylabs.net/contract")
		resp, err := http.Get("https://beta.prylabs.net/contract")
```

It would be nice to log at info here, what do you think?